list/lib/listitem/list-item-only.ts

```diff
@@ -26,4 +26,8 @@ export class ListItemOnly extends ListItemEl {
   override renderRipple() {
     return this.noninteractive ? nothing : super.renderRipple();
   }
+  
```

Comment:
small nit: can you remove the trailing whitespace on this line?

Reply:
Sure!

Reply:
@asyncLiz  I think that did it

Reply:
Thanks!